### DISCUSSION TITLE
Linking to internal achor links (hash links)

### DISCUSSION THREAD

**Discussion by joshlk** [cotes2020/jekyll-theme-chirpy#1755] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I want to link to anchors (hash links) in other pages. For example I want to include the link`/posts/example/#tile1`. When I try to do this I get this error from HTML-Proofer: 
```
  internally linking to /posts/example/#tile1; the file exists, but the hash 'tile1' does not
```
I have double-checked, and that anchor does exist. `/posts/example#tile1` also doesn't work.

What I have discovered is the HTML generated is actually `posts/example/index.html` and so a valid link is `/posts/example/index.html#tile1`. The problem with doing this is then the web browser sees `/posts/example` and `/posts/example/index.html` as two different pages which isn't ideal. Whats the best way of dealing with this?I want to link to anchors (hash links) on other pages.

### Operations you have already tried

above

**Comment by NichtsHsu** [comment, marked as answer]
You might try using liquid syntax to link other posts instead of hard links, i.e., from `[title1](/posts/example/#title1)` to `[title1]({{ site.baseurl }}{% link _posts/xxxx-xx-xx-example.md %}#title1)`.

**Reply by joshlk** [reply]
It doesn't work when I includ the `{{ site.baseurl }}` bit but doing just `[title1]({% link _posts/xxxx-xx-xx-example.md %}#title1)` seems to work fine. Thanks 🙂
